cmd/config/testdata/local_config

```diff
@@ -1,6 +1,9 @@
-api-url: local.api.screwdriver.com
-store-url: local.store.screwdriver.com
-token: local.sd-token
-launcher:
-  version: stable
-  image: screwdrivercd/launcher
+configs:
+  default:
+    api-url: local.api.screwdriver.com
+    store-url: local.store.screwdriver.com
+    token: local.sd-token
+    launcher:
+      version: stable
+      image: screwdrivercd/launcher
+current: default
```

Comment:
This is a workaround to avoid failing tests.

Tests for `sd-local config view` will be modified by another PR.  (same as above)
I think it will be removed because `--local` option will be removed.